watertap/unit_models/tests/test_translator_adm1_asm1.py

```diff
@@ -0,0 +1,303 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
```

Comment:
Update to latest copyright header: https://github.com/watertap-org/watertap/pull/936